homework/12_B/HW01points.py

```diff
@@ -0,0 +1,48 @@
+import math
```

Comment:
This file is in an incorrect location. Move it to
`homework/12_B/<number>_<name>/homework01/HW01points.py`